@kiva/kv-components/vue/KvTextInput.vue

```diff
@@ -45,6 +45,16 @@
 				:icon="mdiAlertCircleOutline"
 				class="tw-absolute tw-top-1.5 tw-right-1.5 tw-pointer-events-none tw-text-danger"
 			/>
+			<button
+				v-if="canClear && valid && !!valueInput"
+				type="button"
+				class="tw-absolute tw-top-1.5 tw-right-1.5"
+				@click="clearInput"
+			>
+				<kv-material-icon
+					:icon="mdiClose"
+				/>
```

Comment:
Since the icon will not announce to screenreaders, please add a `<span class="tw-sr-only">clear input</span>` inside this button element so they can understand what this button does.

Reply:
The span was added.